.github/workflows/test-scraper.yml

```diff
@@ -0,0 +1,103 @@
+# Reference
+# https://github.com/CouncilDataProject/cookiecutter-cdp-deployment
+
+name: Test Scraper
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  test-scraper:
+    if: ${{ startsWith(github.event.comment.body, '/test-scraper') }}
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      #########################################################################
+      # Check initiator is a member of CDP
+
+      - name: Get CDP Organization Members
+        id: cdp-members
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        run: |
+          members="$(gh api -X GET 'orgs/CouncilDataProject/members' -F per_page=100 --paginate --cache 1h --jq '[.[].login] | join("---")')"
```

Comment:
May I ask, why this line doesn't require `gh auth login` of some kind preceding it?

Reply:
I answered myself. The environment variable `GITHUB_TOKEN` is used. The problem was, apparently when I run the `gh` step in my repo for testing, the action doesn't have a `secrets.ACCESS_TOKEN`, only the usual `secrests.GITHUB_TOKEN`. And since the environment variable `GITHUB_TOKEN` gets set to `secrests.GITHUB_TOKEN` (empty), the `gh` call failed with auth error.

So I presume when this is called from this repo, it will indeed have `secrets.ACCESS_TOKEN`... 🤷

Reply:
Will give permissions to this repo to use ACCESS_TOKEN! Thanks for the heads up.